optimum/onnxruntime/modeling_seq2seq.py

```diff
@@ -528,10 +529,9 @@ def forward(
         if past_key_values is not None:
             # Flatten the past_key_values
             past_key_values = [past_key_value for pkv_per_layer in past_key_values for past_key_value in pkv_per_layer]
-
             # Add the past_key_values to the decoder inputs
-            for i, past_key_value in enumerate(past_key_values):
-                onnx_inputs[f"past_key_values_{i}.1"] = past_key_value.cpu().detach().numpy()
+            for input_name, past_key_value in zip(self.key_value_input_names, past_key_values):
```

Comment:
```suggestion
            for input_name, key_value in zip(self.key_value_input_names, key_values):
```
Maybe the names should be updated since those are not necessarily past key values.

Reply:
Thanks for your review ! Model inputs will always be "past" key/values (computed during the previous generation step). I have let some freedom in `self.key_value_input_names` to not be too restrictive on the ONNX model export, but I think keeping the variable name as `past_key_values` is good for clarity